actionpack/test/controller/base_test.rb

```diff
@@ -147,6 +147,10 @@ def test_response_has_default_headers
   ensure
     ActionDispatch::Response.default_headers = original_default_headers
   end
+
+  def test_inspect
+    assert_match(/#<EmptyController:0x[0-9a-f]+>/, @empty.inspect)
```

Comment:
```suggestion
    assert_match(/\A#<EmptyController:0x\h+>$/, @empty.inspect)
```

Reply:
Thanks for the suggestion @esparta. It's probably better to make the regexp more strict.
Though I usually pair the \A with the similar \z:
```assert_match(/\A#<EmptyController:0x\h+>\z/, @empty.inspect)```

Reply:
Sure, I don't see any problem